specifications.yml

```diff
@@ -1007,6 +1109,123 @@ components:
         * pending -> application has started the automatic processing.
         * new -> application is new and has not yet started the automatic processing.
 
+    manualDecision:
+      type: string
+      enum:
+        - granted
+        - referred
+        - denied
+      description: |
+        Current decision of an application.
+        Possible values are
+        * denied -> application is denied and no manual review is necessary.
+        * referred -> application will go through manual review.
+        * granted -> application is granted and no manual review is necessary.
+
+    termsForCustomerApplication:
+      type: object
+      description: Terms of the application. (Look at the examples on the right to understand when they are required)
+      properties:
+        creditLimit:
+          type: number
+          description: "Credit limit for the specific customer"
+        creditLimitCurrency:
+          type: string
+          description: "Currency for the credit limit"
+        creditInsuranceLimit:
+          type: number
+          description: "Credit limit for the insurance"
```

Comment:
Its not really credit limit for an insurance, its more insurance on the credit limit provided :slightly_smiling_face: So maybe describe it as `Amount of credit limit that is covered by insurance`